child_compassion/model/child_compassion.py

```diff
@@ -116,27 +135,72 @@ def _get_child_from_project(project_obj, cr, uid, ids, context=None):
+        'unsponsored_since': fields.date('Unsponsored since'),
+        # TODO : We store the descriptions in child database since we
+        # imported the descriptions from GP in this field. When all children
+        # will have a new case study fetched from Cornerstone,
+        # we can remove the field from db by removing store=True.
+        'desc_en': fields.related(
+            'case_study_ids', 'desc_en', type='text',
+            string=_('English description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_fr': fields.related(
+            'case_study_ids', 'desc_fr', type='text',
+            string=_('French description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_de': fields.related(
+            'case_study_ids', 'desc_de', type='text',
+            string=_('German description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'desc_it': fields.related(
+            'case_study_ids', 'desc_it', type='text',
+            string=_('Italian description'), store={
+                'compassion.child.property': (
+                    _get_child_from_case_study,
+                    ['desc_en', 'desc_fr', 'desc_de', 'desc_it'],
+                    10)}),
+        'has_desc_fr': fields.function(
+            _has_desc, string='FR', type='boolean', multi=True),
+        'has_desc_de': fields.function(
+            _has_desc, string='DE', type='boolean', multi=True),
+        'has_desc_it': fields.function(
+            _has_desc, string='IT', type='boolean', multi=True),
+        'has_desc_en': fields.function(
+            _has_desc, string='EN', type='boolean', multi=True),
         'case_study_ids': fields.one2many(
             'compassion.child.property', 'child_id', string=_('Case studies'),
             readonly=True, track_visibility="onchange"),
-        'portrait': fields.function(get_portrait, type='binary',
-                                    string=_('Portrait')),
-        'state': fields.selection([
-            ('N', _('Available')),
-            ('D', _('Delegated')),
-            ('I', _('On Internet')),
-            ('P', _('Sponsored')),
-            ('R', _('Waiting new sponsor')),
-            ('F', _('Departed'))], _("Status"), select=True, readonly=True,
+        'pictures_ids': fields.one2many(
+            'compassion.child.pictures', 'child_id',
+            string=_('Child pictures'), track_visibility="onchange"),
+        'portrait': fields.related(

```

Comment:
It gets the first one. I should add in object child.pictures _order = 'date desc' so that the first is the most recent. I will fix it.